Core/Core.php

```diff
@@ -666,7 +666,7 @@ function restore_error_handler () {}
  * was thrown.
  * NULL may be passed instead, to reset this handler to its default state.
  * </p>
- * @return callable|null the name of the previously defined exception handler, or null on error. If
+ * @return callable(Throwable):void|null the name of the previously defined exception handler, or null on error. If
```

Comment:
PhpStorm does not support this syntax (yet) IIRC.

Reply:
I just checked with PhpStorm 2019.1 EAP: it makes it just `callable` and types information is treated as documentation. So looks OK to leave it as is.

Reply:
ok, thanks for the feedback!